src/features/commands/psc.js

```diff
@@ -0,0 +1,248 @@
+  
+  let embed = createPlayerEmbed(playerData).setTitle(username).setFooter({ text: `${uuid}\n/v1/hypixel/skyblockplayer` })
+  await interaction.editReply({ content: " ", embeds: [embed]})
+}
+
+async function handleSkyblockItemCommand(client, interaction) {
+  await interaction.reply("Loading...")
+  // Gets the parameters object
+  const parameters = interaction.options
+  const id = parameters.get("id").value
+  await interaction.editReply(`Loading data for ${id}`)
+  
+  const itemData = await getItem(id)
+
+  if (itemData == null) {
+    await interaction.editReply(`No such item found! (${id})`)
+    return
+  }
+  let embed = createItemEmbed(itemData).setTitle(itemData.name).setFooter({ text: `${id}\n/v1/hypixel/skyblockitem` })
+  await interaction.editReply({ content: " ", embeds: [embed]})
+}
+
+function createItemEmbed(itemData) {
+    const embed = new EmbedBuilder()
+    .setColor(config.color)
+
+  embed.addFields(
+    { name: "Item Information:", value: `Name: ${itemData.name}\nRarity: ${itemData.rarity}\nMaterial: ${itemData.material}\nUnstackable: ${itemData.unstackable}\nNPC Sell Price: ${itemData.npcSell} coins` },
+  )
+
+  if (itemData.lowestBin != 0) {
+    embed.addFields({ name: "Auction Information:", value: `Lowest Bin: ${itemData.lowestBin} coins\nAverage Lowest Bin (24 Hours): ${itemData.averageLowestBin} coins` },)
+  } else {
+    embed.addFields({name: "Auction Information:", value: "Sellable: false"})
+  }
+  if (itemData.bazaarBuy != 0 || itemData.bazaarSell != 0) {
+    embed.addFields({ name: "Bazaar Information:", value: `Buy: ${itemData.bazaarBuy} coins\nSell: ${itemData.bazaarSell} coins\nAverage Bazaar Buy (24 Hours): ${itemData.averageBazaarBuy} coins\nAverage Bazaar Sell (24 Hours): ${itemData.averageBazaarSell} coins` },)
+  } else {
+    embed.addFields({name: "Bazaar Information:", value: "Sellable: false"})
+  }
+
+
+  return embed
+}
+
+async function handleResetPSSCache(client, interaction) {
+  await interaction.reply("Loading...")
+
+  const embed = new EmbedBuilder()
+      .setColor(config.color)
+      .setTitle("Clear Cache")
+      .setFooter({ text: "/v1/pss/middlemanagement/resetpublicdata" })
+  const response = await (await requestPSC(`/v1/pss/middlemanagement/resetpublicdata?key=${process.env.CLEAR_CACHE_KEY}`, interaction.member.user.tag)).text()
+
+  embed.addFields({ name: "Response:", value: `${response}`})
+
+  await interaction.editReply({ content: " ", embeds: [embed] })
+}
+
+function createPlayerEmbed(playerdata) {
+  const skyblockplayer = playerdata.skyblockPlayer
+  const currentProfileId = skyblockplayer.currentProfileId
+
+  let currentProfile = {}
+  for (let i = 0; i < skyblockplayer.profiles.length; i++) {
+    const profile = skyblockplayer.profiles[i]
+    if (profile.profileId == currentProfileId) {
+      currentProfile = profile
+      break
+    }
+  }
+
+  const embed = new EmbedBuilder()
+    .setColor(config.color)
+
+  embed.addFields(
```

Comment:
When using template strings, you can have them in multi lines.
Just replace the \n with an actual linebreak.